modules/bibz39/Makefile

```diff
@@ -0,0 +1,646 @@
+# Makefile.in generated by automake 1.14.1 from Makefile.am.
```

Comment:
@PXke you should not commit the `Makefile(.in)` files, but only `Makefile.am`. The former are generated from the latter by `automake` stack.